### PR TITLE
added german readme

### PR DESCRIPTION
**Pull request by CarstenKoenig**
please check the changes I added to the other ones - not sure if I translated and inserted the "german" part correctly

### CODE REVIEW

File: README.ru.md [hunk 1]
```diff
@@ -12,9 +12,7 @@ https://haskell-learning.slack.com/
 Также у нас имеется несколько групп:
 
 - [#learning-lens](https://haskell-learning.slack.com/messages/learning-lens/) - группа посвященная изучению концепции линз и [библиотеке lens](https://hackage.haskell.org/package/lens) в частности.
-- [#haskellbook](https://haskell-learning.slack.com/messages/haskellbook/) - группа чтения [Haskell Book](http://haskellbook.com/).
-
-# Как я могу присоединиться?
+- [#haskellbook](https://haskell-learning.slack.com/messages/haskellbook/) - груb
```
Comment (rasendubi) [inline README.ru.md]: Please, restore this part.

File: README.uk.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ Haskell.
 
 ## Якими мовами ви володієте?
 
-Наші ментори володіють Українською, Російською та Англійською.
+Наші ментори володіють Українською, німецька, Російською та Англійською.
```
Comment (rasendubi) [inline README.uk.md]: Німецькою

File: README.ru.md [hunk 3]
```diff
@@ -35,7 +33,7 @@ https://haskell-learning.slack.com/
 Мы любим Хаскель и хотим разделить эту любовь с другими.
 
 ## Какие языки поддерживаются?
-Среди менторов есть носители русского, украинского и английского языков.
+Среди менторов есть носители русского, немецкий, украинского и английского языков.
```
Comment (rasendubi) [inline README.ru.md]: немецкого

**Comment by CarstenKoenig** [comment]
sorry must have removed some of those by mistake while switching between buffers ... it's hard to see when you don't understand the language

**Comment by rasendubi** [comment]
Great, thanks!

**Pull request merged**